Apps/AdminWebClient/src/ClientApp/src/views/CovidCard.vue

```diff
@@ -70,6 +70,7 @@ export default class CovidCardView extends Vue {
     private isLoading = false;
     private showFeedback = false;
     private showCovidTreatmentAssessment = false;
+    private isSearchIsBlocked = false;
```

Comment:
Nit: this could be simplified to a getter that checks the value of `searchResult.blocked`.  There's also a typo in the name; I would suggest something along the lines of `immunizationsAreBlocked`.